jupyter_book/commands/__init__.py

```diff
@@ -67,37 +75,58 @@ def build(path_book, path_output, config, toc, warningiserror, build):
     if config is not None:
         book_config["yaml_config_path"] = str(config)
         config_yaml = yaml.safe_load(config.read_text())
-
         # Pop the extra extensions since we need to append, not replace
         extra_extensions = config_yaml.pop("sphinx", {}).get("extra_extensions")
+        # Support Top Level config Passthrough
+        # https://www.sphinx-doc.org/en/latest/usage/configuration.html#project-information
+        sphinx_options = ["project", "author", "copyright"]
+        for option in sphinx_options:
+            if option in config_yaml.keys():
+                book_config[option] = config_yaml[option]
 
     # Builder-specific overrides
-    if build == "pdf_html":
-        theme_options = book_config.get("html_theme_options", {})
-        theme_options["single_page"] = True
-        book_config["html_theme_options"] = theme_options
-
-    OUTPUT_PATH = path_output if path_output is not None else PATH_BOOK
-    OUTPUT_PATH = Path(OUTPUT_PATH).joinpath("_build/html")
+    latex_config = None
+    if builder == "pdfhtml":
+        book_config["html_theme_options"] = {"single_page": True}
+    if builder == "pdflatex":
+        if "latex" in config_yaml.keys():
+            latex_config = config_yaml.pop("latex")
+        if "title" in config_yaml.keys():
+            # Note: a latex_documents specified title takes precendence 
+            # over a top level title
+            if latex_config is not None \
+                and "title" not in latex_config['latex_documents'].keys():
```

Comment:
@mmcky , this is the line, where it seems like `latex_documents` tuple is expected in latex_config.

Reply:
ooh. ok my bad. are we planning to pass a `latex_documents` dict with the required keys?